packages/extension/src/view/devtools/components/privateAdvertising/topics/taxonomyTree/tree.js

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// @ts-nocheck
```

Comment:
Is this going to be removed?

Reply:
d3 types give errors to their function/variable definition in many places, so keeping this file in JS. But TS check is still scanning this file, so we need to `@ts-nocheck`.